### PR TITLE
[Feature] New Card Component

### PR DESCRIPTION
**Pull request by arijitdasgupta**
Usage of new card component

```
<Card
  imageSrc="http://pipsum.com/210x150.jpg"
  title="This is a title"
  subtitle="This is a subtitle and this is also a part of the same"
  description={longDescription}
  hightlightText="Highlight"
  tooltipText="Tooltip"
  width="210px"
  height="250px"
/>
```

### CODE REVIEW

File: src/framework/global/_colors.scss [hunk 1]
```diff
@@ -5,8 +5,10 @@ $darkGrey: #354a53;
 $green: #35d0a0;
 $lightGrey: #ebf3f5;
 $lightRed: #f06563;
+$mediumGrey: #7a898e;
 $red: #ff2400;
 $white: #ffffff;
+$ultraLightGrey: #a4a4a4;
```
Comment (mariacasancho) [inline src/framework/global/_colors.scss]: order

Comment (arijitdasgupta) [reply src/framework/global/_colors.scss]: Will fix

Comment (arijitdasgupta) [reply src/framework/global/_colors.scss]: fixed

**Comment by muthan** [comment]
:+1:

**Pull request merged**